README.md

```diff
@@ -25,6 +25,7 @@ brew install <FORMULA>
 | [xray-daemon](https://github.com/aws/aws-xray-daemon) | [formula](Formula/xray-daemon.rb) | Relays traffic to AWS X-Ray API |
 | [aws-simple-ec2-cli](https://github.com/awslabs/aws-simple-ec2-cli) | [formula](Formula/aws-simple-ec2-cli.rb) | CLI tool that simplifies the process of launching, connecting and terminating EC2 instances |
 | [qldbshell](https://github.com/awslabs/amazon-qldb-shell) | [formula](Formula/qldbshell.rb) | A CLI Shell for interacting with and processing queries against AWS QLDB |
+| [amazon-emr-on-eks-custom-image-cli](https://github.com/awslabs/amazon-emr-on-eks-custom-image-cli) | [formula](Formula/emr-on-eks-custom-image.rb) | A CLI tool to interact with EMR on EKS custom images.
```

Comment:
Make sure to change the link text `[amazon-emr-on-eks-custom-image-cli](https://...)` to `[emr-on-eks-custom-image](https://..)`  so that a user can simply `brew install [name]` and get the name from the readme table.

Reply:
Oh, I though it should be the repository name so I didn't change it, let me do that.

Reply:
Done!